packages/react/src/components/Utilities/Icon/index.tsx

```diff
@@ -23,12 +23,33 @@ const iconVariants = cva({
   },
 })
 
+/**
+ * Utility type to extract all possible paths from nested object.
+ * Generates hyphenated paths from nested object structure
+ * Only includes parent key if it has a DEFAULT property
+ */
+type NestedKeyOf<T> = {
+  [K in keyof T & string]: T[K] extends object
+    ? K extends "DEFAULT"
+      ? never
+      : T[K] extends { DEFAULT: string }
+        ? `${K}` | `${K}-${NestedKeyOf<T[K]>}`
+        : `${K}-${NestedKeyOf<T[K]>}`
+    : K extends "DEFAULT"
+      ? never
+      : `${K}`
+}[keyof T & string]
+
 export interface IconProps
   extends SVGProps<SVGSVGElement>,
     VariantProps<typeof iconVariants> {
   icon: IconType
   state?: "normal" | "animate"
-  color?: Lowercase<keyof typeof f1Colors.icon> | `#${string}` | "currentColor"
+  color?:
+    | "default"
```

Comment:
What `default` color does?

Reply:
It was already there in the `Icon` component: 

```tsx
  const getColorClass = (colorValue: string) => {
    if (colorValue === "currentColor") return "text-current"
    if (colorValue === "default") return "text-f1-icon"
    if (colorValue.startsWith("#")) return ""
    return `text-f1-icon-${colorValue}`
  }
```